src/map/script.cpp

```diff
@@ -25677,6 +25677,78 @@ BUILDIN_FUNC( openstylist ){
 #endif
 }
 
+/**
+* Generate <ITEML> string for client
+* itemlink(<item_id>{,<refine>,<card0>,<card1>,<card2>,<card3>,<enchantgrade>});
+* itemlink2(<item_id>,<refine>,<card0>,<card1>,<card2>,<card3>,<enchantgrade>,<RandomIDArray>,<RandomValueArray>,<RandomParamArray>);
+* @author [Cydh]
+**/
+BUILDIN_FUNC(itemlink)
+{
+	struct s_item_link itemldata;
+
+	memset(&itemldata, 0, sizeof(s_item_link));
```

Comment:
```suggestion
	s_item_link itemldata = {};
```
No more memset, please! :(